### DISCUSSION TITLE
import from notion

### DISCUSSION THREAD

**Discussion by vasyakrg** [outline/outline#7799] [Self Hosting]
**To Reproduce**
1. Go to 'Settings > import > notion'
2. See error: Failed • Uploaded file does not contain any valid collections. It may be corrupt, the wrong type, or version.. Check server logs for more details.

Outline
- Install: self hosted
- Version: 0.80.2

exported info from notion in html format by zip file.
on console a have error: 

```
Failed • Uploaded file does not contain any valid collections. It may be corrupt, the wrong type, or version.. Check server logs for more details. 
```

in zip file a have index.html, *.html, csv, png and ico files.
index file in browser work.

**Comment by Cloud11PL** [comment]
I have the same issue

**Comment by paul-tharun** [comment]
I fixed it by unziping the notion export file. Putting all the files in a  new folder and rezipping it. 
earlier zip has only files, now zip has folder_name > files 

The folder_name will become your collection name after import.

**Reply by tommoor** [reply]
It's possible / probable that Notion changed the export format at some point

**Reply by paul-tharun** [reply]
I think this is how the export happens when there are no team spaces, and just the default privateworkspace

**Reply by tommoor** [reply]
I think we can detect this and import into a default "Notion" collection in this circumstance

**Comment by davidbeig** [comment]
I had the same problem just today, turns out the .zip file provided by notion has inside another .zip file, that's the one we have to import in outline.
